lib/SPIRV/libSPIRV/SPIRVFunction.h

```diff
@@ -171,6 +171,36 @@ class SPIRVFunction : public SPIRVValue, public SPIRVComponentExecutionModes {
 
 typedef SPIRVEntryOpCodeOnly<OpFunctionEnd> SPIRVFunctionEnd;
 
+
+class SPIRVConstFunctionPointerINTEL : public SPIRVValue {
```

Comment:
I wonder, why this is not a `SPIRVInstruction` anymore?

Reply:
Due to the same reason why other OpConstant* are not as well